### ISSUE TITLE
Change MaterialSublotProperty element to MaterialLotProperty for MaterialSubLotType in Material.xsd to align with updated 950002

### ISSUE THREAD

**Issue by racheliurui** [ETPartners/b2mmlv7beta#34]
### MESA
https://github.com/MESAInternational/B2MML-BatchML/issues/43

### Background 
In Material Model, Per 950002, Both material lot and material sub lot objects apply the same material lot property object and instances in their definition.  Thus in updated ISA-950002, there's no such model called MaterialSublotProperty, 
In Material.xsd / MaterialSublotType, CHANGE MaterialSublotProperty element to MateralLotProperty reflect the property type.

### Supporting Document
>ISA-950002 JWG5 CDV01 version (2019 12)
>Table 72 – Material sublot relationship roles

### Impacted Types
B2MML-Material.xsd

- MaterialSubLotType

**Comment by Dennis-Brandl** [comment]
I do not believe that the element should be MaterialLotProperty,, and should remain as MaterialSublotPropertywith a type of MaterialLotPropertyType.
